packages/node_modules/@ciscospark/bin-sauce-connect/src/start.js

```diff
@@ -80,8 +80,11 @@ function connect() {
 
     spawn(`sc/bin/sc`, [
       `-B`, [
-        `mercury-connection-a.wbx2.com`,
-        `mercury-connection-intb.ciscospark.com`
+        `mercury-connection-intb.ciscospark.com`,
+        `idbroker.webex.com`,
+        `127.0.0.1`,
+        `localhost`,
+        `*.wbx2.com`
```

Comment:
might want to throw `*.ciscospark.com` in here for other stuff in the integration environment